expo/gbe/templates/gbe/landing_page.tmpl

```diff
@@ -1,37 +1,39 @@
 {% extends "base.tmpl" %}
 {% block title %}
-   Profile View  
+   Profile View
 {% endblock %}
 
 {% block content %}
-     
+
 <div class="colmask threecol">
      <div class="colmid">
-	  <div class="colleft">
-	       <div class="col1">
-                    {% if profile and bookings %}
-                    <div class='landing_box'>
-			 {% include 'gbe/incl_schedule.tmpl' %}
-		    </div>
-		    &nbsp;
-		    {% endif %}
-                    {% if profile and tickets %}
-                    <div class='landing_box'>
-			 {% include 'gbe/incl_tickets.tmpl' %}
-		    </div>
-		    {% endif %}
-	       </div>
-	       <div class="col2">
-		    {% include 'gbe/incl_lp_left.tmpl' %}
-	       </div>
-	       <div class="col3">
-                    {% if profile %}
-                    <div class='landing_box'>
-                        {% include 'gbe/incl_lp_right.tmpl' %}
-                    </div>
-                    {% endif %}
-	       </div>
-	  </div>
+       <div class="colleft">
+         <div class="col1">
+            {% if profile and bookings %}
+                <div class='landing_box'>
+                  {% include 'gbe/incl_schedule.tmpl' %}
+        		    </div>
+  	        &nbsp;
+  	        {% endif %}
+
+            {% if profile and tickets %}
+            <div class='landing_box'>
+        			 {% include 'gbe/incl_tickets.tmpl' %}
+               {% include 'gbe/profile/incl_performer.tmpl' %}
```

Comment:
I think I found my problem - I have a user that has a profile but who has not purchased tickets for the current conference.  Because this is in the "if profile and tickets" clause, the profile box only shows up if I also have tickets.  

This is actually a really frequent case - teachers come in to propose classes before they buy tickets - if they get their classes accepted, they use the teacher discount they get to buy cheaper tickets.